packages/explorer/src/helpers/postMessageRelayHelpers.ts

```diff
@@ -383,7 +385,7 @@ export async function executeOperation({
                 ? {
                     message: err.message,
                     ...('stack' in err && typeof err.stack === 'string'
```

Comment:
If the error stack should be in extensions, is [this wrong](https://github.com/apollographql/embeddable-explorer/blob/1a5713028b5510748d0ece50d8e61568772a381b/packages/sandbox/src/helpers/readMultipartWebStream.ts#L24)? [and this](https://github.com/apollographql/embeddable-explorer/blob/1a5713028b5510748d0ece50d8e61568772a381b/packages/sandbox/src/helpers/readMultipartWebStream.ts#L11)?

I can't find where the extensions wrapper comes in in the multipart response spec [here](https://github.com/apollographql/router-private/blob/subscriptions/dev-docs/multipart-subscriptions-protocol.md). 

If you test this and errors work, that works for me. 

Did we test with defer & subscriptions?

Reply:
Yes, those are incorrect too then.

The multipart spec has more wording around the streaming and chunking bits. It just says to follow the GraphQL spec for the other types

> Both types of errors will follow the [GraphQL error format](http://spec.graphql.org/draft/#sec-Errors.Error-Result-Format) (but top-level errors will never have locations or path).

That spec states that the GraphQL Error type should only have 4 fields, `message`, `path`, `locations`, and `extensions`. Any other field (like `stack`) is _discouraged_ and should instead live under `extensions`

> Previous versions of this spec did not describe the extensions entry for error formatting. While non-specified entries are not violations, they are still discouraged.

Reply:
curious if we were able to test that the error messages are getting through as expected w stack. Can we update this PR to edit the multipart & subscription types to wrap in extension?

Reply:
Stack message still pass through as expected today, but we are technically not compliant